src/SingleTableInheritanceTrait.php

```diff
@@ -171,15 +171,16 @@ public function setSingleTableType() {
    */
   public function newFromBuilder($attributes = array(), $connection = null) {
     $typeField = static::$singleTableTypeField;
+    $attributes = (array) $attributes;
```

Comment:
if we put the (array) cast in line 172 does this cast happen automatically? If so lets do that.

Reply:
You can't cast in a signature, you can only type hint, and type hinting will throw an error not cast when it receives something other than an array. Had type hinting been in place here on Model this issue would've been caught when the library was written though haha.